plugins/catalog-backend/src/search/DefaultCatalogCollator.ts

```diff
@@ -81,6 +81,24 @@ export class DefaultCatalogCollator implements DocumentCollator {
     return formatted.toLowerCase();
   }
 
+  private isUserEntity(entity: Entity): entity is UserEntity {
+    return entity.kind === 'User';
```

Comment:
`kind` should be case in-sensitive. So we should lower/upper case both for comparison.